setup.cfg

```diff
@@ -10,10 +10,9 @@ long_description_content_type = text/x-rst
 classifiers =
     Development Status :: 5 - Production/Stable
     License :: OSI Approved :: Apache Software License
-    Programming Language :: Python :: 2.7
-    Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
```

Comment:
Can we add 3.10, 3.11, 3.12, or do we depend on epicscorelibs?  Probably not a question for this PR.